#! /bin/sh /usr/share/dpatch/dpatch-run
## 50_mountoverflowtmp-localfs.dpatch by Petter Reinholdtsen

   * Add mountoverflowtmp to virtual facility $local_fs, to make sure
     /tmp/ is always mounted before the scripts needing /tmp/ is
     started.  Conflict with initscripts before 2.86.ds1-48, to avoid a
     dependency loop with this change.

@DPATCH@
Index: debian/control
===================================================================
--- debian/control	(revision 470)
+++ debian/control	(working copy)
@@ -11,6 +11,7 @@
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, sysv-rc
 Suggests: bootchart
+Conflict: initscripts (<< 2.86.ds1-48)
 Description: Reorder boot sequence based on LSB init.d script dependencies
  Reorder the boot order based on init.d script dependencies as
  documented in LSB comment headers in the script or in override files
Index: debian/patches/31_debian_conf.dpatch
===================================================================
--- debian/patches/31_debian_conf.dpatch	(revision 470)
+++ debian/patches/31_debian_conf.dpatch	(working copy)
@@ -15,7 +15,7 @@
  # All local filesystems are mounted (done during boot phase)
  #
 -$local_fs	boot.localfs
-+$local_fs	+mountall +umountfs
++$local_fs	+mountall +mountoverflowtmp +umountfs
  
  #
  # Low level networking (ethernet card)
